build/tsnode.js

```diff
@@ -48,6 +49,25 @@ module.exports = function initTsNode() {
 		addMatchAll: false
 	});
 
+	const
+		// @ts-ignore
```

Comment:
А почему игнор? Что тут ты хочешь сделать?

Reply:
Игнор потому что ts не подсвечивает что есть метод resolveFilename и кричит

Reply:
Но ведь это не правильно. Мы столько усилий прикладываем чтобы избавиться от этих игноров. Тем более что тут они не нужны.

Я бы сделал вот так:

```typescript
{
	console.log('`Module._resolveFilename` will be locked and cannot be overwritten in the future');

	const
		resolver = Module['_resolveFilename'];

	// Sets a hook to prevent overwriting the path resolver function.
	// For example, PW overrides `_resolveFilename` with its own version, which does not work correctly.
	// Unfortunately, PW does not have any option to disable this behavior.
	Object.defineProperty(Module, '_resolveFilename', {
		get() {
			return resolver;
		},

		set() {
			// ...
		}
	});
}
```